ykaddr/src/obj.rs

```diff
@@ -6,8 +6,6 @@ use libc::c_void;
 use libc::{
     Elf64_Addr as Elf_Addr, Elf64_Off as Elf_Off, Elf64_Word as Elf_Word, Elf64_Xword as Elf_Xword,
 };
-use memmap2;
```

Comment:
The only redundant imports I don't understand are these ones. :man_shrugging:

Reply:
I've noticed that you don't need to import top-level crates any more. Once they are in Cargo.toml, they are in scope.